probe/tag/docker_tagger.go

```diff
@@ -157,15 +179,15 @@ func (t *DockerTagger) update() bool {
 	}
 }
 
-func (t *DockerTagger) updateContainers(client dockerClient) error {
-	apiContainers, err := client.ListContainers(docker.ListContainersOptions{All: true})
+func (t *DockerTagger) updateContainers() error {
+	apiContainers, err := t.client.ListContainers(docker.ListContainersOptions{All: true})
 	if err != nil {
 		return err
 	}
 
 	containers := []*docker.Container{}
```

Comment:
To make it retry if it fails; if a connection is dropped for instance, or Docker is restarted.